dev/scripts/add_test_users.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+echo "running $0 from directory $(pwd)"
+
+# Credentials for Basic Auth
+USERNAME="SDP-admin"
+PASSWORD="api_key_1234567890"
+
+# Retrieve tenant details using curl with Basic Auth from an API endpoint
+TENANTS_JSON=$(curl -u $USERNAME:$PASSWORD -s http://localhost:8003/tenants/)
+
+if [ -z "$TENANTS_JSON" ]; then
+  echo "No tenant details could be retrieved. Exiting."
+  exit 1
+fi
+
+# Function to add a user for  tenant
+add_user_for_tenant() {
+  echo
+  local tenant_id=$1
+  local tenant_name=$2
+  local email="owner_${tenant_name}@example.com"
```

Comment:
This value here doesn't match with the examples in the README. shouldn't the emails be `xxx@$tenant.local` ?

Reply:
These values appear unused too. `email`, `username` and `password`